website/src/components/Icons/IconColor.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { getIconProps } from '@pluralsight/headless-styles'
+import { WarningIcon } from '@pluralsight/icons'
+import Container from '../Container/Container'
+
+const psIconProps = getIconProps()
+
+function IconColor() {
+  return (
+    <Container>
+      <div
+        style={{
+          color: 'var(--ps-danger-text)',
+          backgroundColor: 'var(--ps-danger-background-weak)',
+          display: 'flex',
+          alignItems: 'center',
+          padding: '1em',
+          borderRadius: '3px',
+          width: '100%',
+        }}
+      >
+        <WarningIcon {...psIconProps} />
+        &nbsp; This is a warning!
```

Comment:
Copywriting 101 - never ever use "!". We don't ever want to yell at people. 😉 Instead, try to match the new brand by being more holistic and human.

```suggestion
        &nbsp; Watch for demogorgons.
```